packages/auth/migrations/20230907201006_rename-limits-jsonb-keys.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex) {
+  // renames sendAmount keys (if any) to debitAmount in limits jsonb column
+  return knex('accesses')
+    .update({
+      limits: knex.raw(
+        "limits - 'sendAmount' || jsonb_build_object('debitAmount', limits->'sendAmount')"
+      )
+    })
+    .whereRaw("limits \\? 'sendAmount'")
+}

```

Comment:
This is hard to read but I could not find a simpler way.

To test I initialized rafiki with this extra test migration (before `20230907201006_rename-limits-jsonb-keys.js`)  which seeds some dummy data: https://gist.github.com/BlairCurrey/094b5df41f36f94befacbf727c3535ef

Then I verified the accesses records matched expectations after rebuilding rafiki from migrations.